### PR TITLE
chore: use Lucide icons in ToggleButton and clean up styles

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
We were just using a `i` as an icon.

Closes: AVO-440

## Description

Switches to use an actual icon and cleans up some of the styles.

#### On master:
image
image

#### In this branch:
image
image

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/components/ToggleButton.tsx [hunk 1]
```diff
@@ -79,16 +80,16 @@ export default function ToggleButton({
                 role="button"
                 tabIndex={0}
                 className={twMerge(
-                  'ml-2 inline-flex h-[29px] w-[29px] select-none justify-center self-center rounded-full bg-white shadow dark:border dark:border-gray-500/80 dark:bg-gray-600/50',
+                  'inline-flex select-none items-center justify-center',
                   isToolTipOpen && 'pointer-events-none'
                 )}
               >
-                <div className="self-center text-xs font-bold">i</div>
+                <Info className="text-gray-500" />
```
Comment (silkeholmebonnen) [inline web/src/components/ToggleButton.tsx:87]: I think the icon should be a bit bigger
```suggestion
                <Info className="text-gray-500" size={30} />
```

Comment (VIKTORVAV99) [reply web/src/components/ToggleButton.tsx:87]: I considered it but then it felt too big compared to the other info icons (the one in the zone header in particular) but we can try bumping it a bit and see if you wish?

Comment (VIKTORVAV99) [reply web/src/components/ToggleButton.tsx]: Now it looks like this:
image


Not a huge fan of it as now the icons look miss aligned and too big compared to the rest?

Comment (silkeholmebonnen) [reply web/src/components/ToggleButton.tsx]: I prefer it as it is right now as it aligns with the outer circle. If you want to we can probably get alex take on it tomorrow ?

Comment (VIKTORVAV99) [reply web/src/components/ToggleButton.tsx]: Sounds good, there is no hurry here so it's better we take our time with it.

Comment (VIKTORVAV99) [reply web/src/components/ToggleButton.tsx]: I just saw Alex had made a prototype design for this so I'll go ahead and implement that: 
image

Should fix both our concerns I think.

**Comment by VIKTORVAV99** [comment]
@Alportan could you double check this. It kind of pops a little too much on darkmode but the gray-500 don't have enough contrast.

**Comment by Alportan** [comment]
> @Alportan could you double check this. It kind of pops a little too much on darkmode but the gray-500 don't have enough contrast.

Hmm, I agree. It does stand out too much. 
image

I would suggest the following icon colours:
🌝 Light mode: `gray-300`
🌚 Dark mode: `neutral-500`

**Review by Alportan — APPROVED** [review]
Looks good on my side! ✨

We might revise it later when we make a decision on the elevation colours for backgrounds (the blur effect) to keep the colour consistent with the info icon on the side panel. 🙌

**Comment by VIKTORVAV99** [comment]
> Looks good on my side! ✨
> 
> We might revise it later when we make a decision on the elevation colours for backgrounds (the blur effect) to keep the colour consistent with the info icon on the side panel. 🙌

Sounds good, and revising later is not a problem, in fact I'd be happy to do it so we have a consistent and unified design!

**Comment by VIKTORVAV99** [comment]
@silkeholmebonnen does this look good to you as well?

**Review by silkeholmebonnen — APPROVED** [review]
LGTM

**Pull request merged**